plugins/module_utils/mh/mixins/cmd.py

```diff
@@ -164,4 +164,11 @@ def run_command(self, extra_params=None, params=None, *args, **kwargs):
         options.update(kwargs)
         rc, out, err = self.module.run_command(self.vars.cmd_args, *args, **options)
         self.update_output(rc=rc, stdout=out, stderr=err)
-        return self.process_command_output(rc, out, err)
+        if process_output is None:
+            _process = self.process_command_output
+        elif isinstance(process_output, str):
+            _process = getattr(self, process_output)
```

Comment:
Is this case really needed? Such magic of supplying a function *name* is something I don't really like, since it defies any kind of static analysis :)

Reply:
Well... *really* needed, no, it is not. And now on a second though, the developer could directly pass a reference to the method instead of just the plain name. Yeah, this is overkill, removing it.